pkg/acloudapi/apitypes.go

```diff
@@ -152,6 +153,8 @@ type CreateCluster struct {
 	NodePools   []NodePools         `json:"nodePools" yaml:"NodePools"`
 	IPWhitelist []IPWhitelistEntry  `json:"ipWhitelist,omitempty" yaml:"IpWhitelist,omitempty"`
 	Addons      map[string]APIAddon `json:"addons,omitempty" yaml:"Addons,omitempty"`
+
+	AutoScalerSettings AutoscalingSettings `json:"clusterAutoscalerSettings,omitempty" yaml:"ClusterAutoScalerSettings,omitempty"`
```

Comment:
This should be a pointer 

```suggestion
	AutoScalerSettings *AutoscalingSettings `json:"clusterAutoscalerSettings,omitempty" yaml:"ClusterAutoScalerSettings,omitempty"`
```